src/Lykke.Job.CandlesHistoryWriter.Services/Candles/CandlesCacheInitalizationService.cs

```diff
@@ -51,16 +51,16 @@ public class CandlesCacheInitalizationService : ICandlesCacheInitalizationServic
             await _log.WriteInfoAsync(nameof(CandlesCacheInitalizationService), nameof(InitializeCacheAsync), null, "All candles history is cached");
         }
 
-        private async Task CacheAssetPairCandlesAsync(IAssetPair assetPair, DateTime toDate)
+        private async Task CacheAssetPairCandlesAsync(IAssetPair assetPair, DateTime now)
```

Comment:
Do we actually need to put 'now' into parameters? Could we call +clock.UtcNow in method body instead? Do I get it right that the idea is to keep 'now' without changing for different asset pairs while caching?

Reply:
Yes, idea is to keep now unchanged